src/app/main.cpp

```diff
@@ -172,19 +172,30 @@ int main( int argc, char **argv )
 #if defined( Q_OS_ANDROID )
   QString projPath = PlatformUtilities::instance()->systemGenericDataLocation() + QStringLiteral( "/proj" );
   qputenv( "PROJ_LIB", projPath.toUtf8() );
-  QgsApplication app( argc, argv, true, PlatformUtilities::instance()->systemGenericDataLocation() + QStringLiteral( "/qgis/resources" ), QStringLiteral( "mobile" ) );
```

Comment:
Why don't you move this logic to platformutils?

Reply:
@suricactus , I felt it wasn't needed since neither Linux nor Windows are in need of this.

Reply:
I was more because (almost) the same code is a few lines below. But if you say it is not needed, you are more familiar with the change. Feel free to merge.

Reply:
@suricactus , right, for the iOS lines, it might further change based on @3nids ' own testing and findings.